package.json

```
@@ -1,14 +1,14 @@
 {
-  "name": "node-core-test",
+  "name": "test",
   "version": "2.0.0",
-  "description": "Node 18's node:test, as a node module",
+  "description": "Node 18's node:test, as a npm package",
   "license": "MIT",
-  "repository": "juliangruber/node-core-test",
+  "repository": "nodejs/node-core-test",
   "main": "./index.js",
   "types": "./index.d.ts",
   "bin": {
     "node--test": "./bin/node--test.js",
-    "node-core-test": "./bin/node-core-test.js"
+    "test": "./bin/test.js"
```

I'm not sure we should rename that file, npm doesn't use it anyway, and `test.js` is not a great name for a program entry point imo.
```suggestion
    "test": "./bin/node-core-test.js"
```

---

Hm ok make sense